campaign/controllers.py

```diff
@@ -1295,8 +1295,9 @@ def create_campaignx_supporters_from_positions(
     success = True
     # timezone = pytz.timezone("America/Los_Angeles")
     # datetime_now = timezone.localize(datetime.now())
-    # date_today_as_integer = convert_date_to_date_as_integer(datetime_now)
-    date_today_as_integer = get_current_date_as_integer()
+    datetime_now = generate_localized_datetime_from_obj()[1]
+    date_today_as_integer = convert_date_to_date_as_integer(datetime_now)
```

Comment:
Hi @pbnjcub, thank you for your changes. I want to leave get_current_date_as_integer() as a helper function because I don't need to bring the complexity of generate_localized_datetime_from_obj out to these locations where we just need: `date_today_as_integer = get_current_date_as_integer()`
I will roll this back and we can discuss in the team meeting.

Reply:
ok looking forward to discussing tomorrow.